java-dynamic-sqs-listener-core/src/main/java/com/jashmore/sqs/util/Suppliers.java

```diff
@@ -0,0 +1,66 @@
+package com.jashmore.sqs.util;
+
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.UtilityClass;
+
+import java.util.function.Supplier;
+
+/**
+ * Provides helper methods for building {@link Supplier}s.
+ *
+ * <p>This has been implemented again to reduce dependencies on other modules, more specifically Guava.
```

Comment:
Yeh at this point I feel really strongly about pulling Guava in and removing this code. My 2c.

Reply:
See comment above but I am happy to do that